src/hotspot/share/gc/shared/referenceProcessor.hpp

```diff
@@ -48,14 +48,14 @@ class RefProcProxyTask;
 class EnqueueDiscoveredFieldClosure {
 public:
   // For the given j.l.ref.Reference reference, set the discovered field to value.
-  virtual void enqueue(oop reference, oop value) = 0;
+  virtual void enqueue(HeapWord* discovered_field_addr, oop value) = 0;
```

Comment:
Comments hare a slightly out of date now since passing in the address to the field.